scss/components/_badge.scss

```diff
@@ -47,7 +51,7 @@ $badge-font-size: 0.6rem !default;
       @if $name != primary {
         &.#{$name} {
           background: $color;
```

Comment:
I'm not sure about this. The `$foundation-palette` is a list of colors, not a list of backgrounds. We're using a `$color` from the palette to set a `background`. Seems logical as-is to me.